.prettierrc

```diff
@@ -0,0 +1,4 @@
+{
+  "printWidth": 80,
+  "singleQuote": false
+}
```

Comment:
@stephan281094 Needed to add this file back because without I was getting conflicting errors. VSCode would gives errors on all the double quotes and then format it to single quotes and then conflict with something else, not sure if that made sense 🙃  Looks like my global VSCode prettier settings had single quote somehow
<img width="1065" alt="Screen Shot 2022-10-13 at 3 47 33 PM" src="https://user-images.githubusercontent.com/39889198/195697079-309bd1a0-6d2d-42bc-95a0-0891cf3b7c5c.png">